### ISSUE TITLE
Can't create a multisig wallet with BitBox 02

### ISSUE THREAD

**Issue by bilthon** [cryptoadvance/specter-desktop#815]
I have a Ledger Nano S and a BitBox02. I can register both devices without any issues, but when it comes to create a multisig wallet I can only see the Ledger as an available device.

Is this a known issue? any tips on how to overcome it? This is v0.10.4 on mac.
![Screen Shot 2020-12-27 at 11 14 58 PM](https://user-images.githubusercontent.com/1840721/103189187-767c8400-4899-11eb-9224-60313759d1e9.png)

**Comment by ben-kaufman** [comment]
Hi, we do not yet support BitBox02 for multisig, but it's under work currently and should soon be possible.

**Comment by bilthon** [comment]
Cool, I'm happy to help with the tests if needed then. Is there a specific branch on which this work is being done?

**Comment by ben-kaufman** [comment]
Not really, most of the work is at https://github.com/bitcoin-core/HWI/pull/407
Once HWI stuff is working well it's quite fast to add it to Specter.

**Comment by k9ert** [comment]
Closing this for now.

**Comment by nioncode** [comment]
@k9ert can we re-open this until the feature is implemented so that we can subscribe to this issue to be notified when it is done?
